ti/instrumentation/ruby/helper.go

```diff
@@ -83,26 +83,26 @@ func UnzipAndGetTestInfo(agentInstallDir string, log *logrus.Logger) (scriptPath
 	return scriptPath, nil
 }
 
-func WriteGemFile(workspace, repoPath string) error {
-	pattern := fmt.Sprintf("%s/**/Gemfile", workspace)
+func AddHarnessRubyAgentToGemfile(repoPath string, log *logrus.Logger) error {
+	cmd := exec.Command("bundle", "add", "harness_ruby_agent", "--path", fmt.Sprintf("%q", repoPath), "--version", "~> 0.0.1") // #nosec
```

Comment:
This seems to assume bundle is installed on the VM/POD and is in the $PATH, right?

Reply:
Will this also install the gem or just add to the Gem file?

Reply:
Does this fail the build if pwd is not a ruby repo? Should we document user need cd into the corresponding folder that contains the ruby stuff in pre-command?